css-three-column-layout/styles.css

```diff
@@ -0,0 +1,74 @@
+* {
+  font-family: Lato, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+}
+
+h1 {
+  text-align: center;
+  color: rgb(255, 255, 255);
+  width: 100%;
+  padding-top: 50px;
+  padding-bottom: 25px;
```

Comment:
If you're using this to add space between the `h1` & the content below, you should be using `margin-bottom` to space it out instead. `padding` is used to space content around it's border, whereas `margin` is used to space out elements between each other